Userland/Libraries/LibWeb/Layout/LayoutState.cpp

```diff
@@ -334,7 +334,7 @@ void LayoutState::UsedValues::set_node(NodeWithStyleAndBoxModelMetrics& node, Us
 
 void LayoutState::UsedValues::set_content_width(CSSPixels width)
 {
-    VERIFY(isfinite(width.to_double()));
+    VERIFY(!width.might_be_saturated());
```

Comment:
Is this even a problem anymore though? Sure the layout is probably wrong, but we don't have the problems associated with infinity anymore right?